uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java

```diff
@@ -273,7 +273,7 @@ public void handleAppProtectionPolicy() {
         try {
```

Comment:
Do we need to try-catch now, or can that be handled at the calling method.

Reply:
better to just do it here and fail the test right away if UI Element is not found

Reply:
I think that's fine. 
nit: This comment was a general comment for other changes as well, in case we want to handle any kind of exception and not just the one that is being caught